### ISSUE TITLE
Parser fails when tabs are used for whitespace

### ISSUE THREAD

**Issue by kliuless** [rmosolgo/graphql-ruby#144]
Fails on 5977fe27 (note the \t):
```
  let(:query_string) {%|
    query getDirectives {
      __schema {
        \tdirectives { name, args { name, type { name, ofType { name } } }, locations }
      }
    }
  |}
```
Error is: `GraphQL::ParseError: Unexpected end of document`

Works fine on v0.12.0.

**Comment by rmosolgo** [comment]
Oh! Thanks for reporting that. It should be fixed by `0aec898`, can you try it again?

**Comment by kliuless** [comment]
Looks good, thanks.
